README.md

```diff
@@ -54,6 +54,85 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Group Files
```

Comment:
Group is used it seems for the top level commands in the apiary, and group files is already taken from the mfs group, nomenclature ftw! we should talk more on what we call commands in ipfs in general.